### PR TITLE
Print message if no logs for instance

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Fixes https://github.com/giantswarm/cli/issues/41 

If there are no logs at all for this instance, print a quietable message.

```
Josephs-MacBook-Pro:cli joseph$ ./swarm create ./int-tests/apps/beep/swarm.json
Creating 'beep' in the 'salisbury_joe/testing' environment…
Service created successfully!
Josephs-MacBook-Pro:cli joseph$ ./swarm status beep
Service 'beep' is down

component            image     instanceid    created              status
beep-service/beeper  python:3  chykokc8vbi2  2015-11-27 14:59:27  down
Josephs-MacBook-Pro:cli joseph$ ./swarm logs chykokc8vbi2
Currently, no log entries are available for this instance.
Josephs-MacBook-Pro:cli joseph$ ./swarm logs -q chykokc8vbi2
Josephs-MacBook-Pro:cli joseph$
```

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
ping @giantswarm/team-features

File: int-tests/029-no-logs-message.t [hunk 1]
```diff
@@ -0,0 +1,10 @@
+No logs:
+
+  $ swarm -q delete --yes beep 2>&1 > /dev/null
```
Comment (ZeissS) [inline int-tests/029-no-logs-message.t]: do the other tests also delete this first?

File: logs.go [hunk 2]
```diff
@@ -132,7 +133,14 @@ func LogInstance(arg string, logsTail logsTailFlags, logsFollow bool) {
 	}
 	defer resBody.Close()
 
-	if _, err := dockerutils.StdCopy(os.Stdout, os.Stderr, resBody); err != nil {
+	buffer := new(bytes.Buffer)
+	buffer.ReadFrom(resBody)
```
Comment (ZeissS) [inline logs.go]: does this work with following logs? :P

File: int-tests/029-no-logs-message.t (hunk 1 above)
Comment (JosephSalisbury) [inline int-tests/029-no-logs-message.t]: copied from 007-cat.t, so yes - few others do it as well

**Comment by ZeissS** [comment]
fine with me - @marians ?

**Comment by marians** [comment]
Yep, LGTM

File: int-tests/029-no-logs-message.t (hunk 1 above)
Comment (ewoutp) [inline int-tests/029-no-logs-message.t]: This is indeed intended.

**Comment by marians** [comment]
Stupidly I just tested this now and found that the new message is sent to STDOUT. So when piping the log into a file, the message is the only content.

I suggest either printing to STDERR or only printing if `isTerminal` is true.

**Comment by ZeissS** [comment]
+1 printing when isTerminal to stderr ;)

**Pull request merged**